packages/vaex-core/vaex/dataframe.py

```diff
@@ -5747,6 +5751,40 @@ def export_fits(self, path, column_names=None, shuffle=False, selection=False, p
         import vaex.export
         vaex.export.export_fits(self, path, column_names, shuffle, selection, progress=progress, virtual=virtual, sort=sort, ascending=ascending)
 
+    def export_csv(self, path, virtual=True, selection=False, progress=None, batch_size=1_000_000, **kwargs):
+        """ Exports the DataFrame to a CSV file.
+
+        :param str path: Path for filename
+        :param bool virtual: If True, export virtual columns as well
```

Comment:
The default value for virtual is False for the others, should we remove it altogether. Or change all the other default to True?

Reply:
Ah my bad, i was testing it and forgot the switch it back. I say for now let's keep everything consistent so virtual=False by default indeed. Thanks!